bot/cogs/snakes.py

```diff
@@ -40,6 +69,28 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        embed = discord.Embed(color=0x3E885B)
+        if name == "python":
```

Comment:
This should be case insensitive.

Reply:
Thanks - so when I change line 73 to `if name.lower() == "python":`, I get an error about comparing NoneTypes since the default name is `None` (see the method declaration). How should I get around this? Should I do an "if None" check before the python special case or should I do something else?

(`discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'lower'` is the error for reference)

Reply:
```py
if name and name.lower() == "python":
```
would work for me.